### DISCUSSION TITLE
T3 with Honor Points system

### DISCUSSION THREAD

**Discussion by unitscan** [chromiecraft/chromiecraft#4491] [Suggestions]
With the opening of the 60-64 bracket, a good idea would be make the T3 set available for Honor Points, in order to incentivize PVP even more

**Comment by Santaria94** [comment]
No it's a bad idea

**Comment by Amrx2** [comment]
Hi unitscan, I can confirm we will not be implementing this. We are figuring out a system so players can, at the very least, get T3 transmogs itself. 

Have a good day,
Construct

**Reply by unitscan** [reply]
Actually I would be more interested in the T3  transmog set than the gear itself. It would be a good idea to give the option to make available only the set skin.

**Comment by 55Honey** [comment]
Hi unitscan,
Thanks for your suggestion.
Giving a PvE set for PvP activity only, which would be BiS for 2 brackets at least, is certainly not going to happen.

**Reply by unitscan** [reply]
Actually I would be more interested in the T3 transmog set than the gear itself. It would be a good idea to give the option to make available only the set skin.
